lib/index-basic.js

```diff
@@ -0,0 +1,19 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+var Core = require('./core');
```

Comment:
(To clarify, I had to stop and think whether core pulled it in or whether it was something pulled in indirectly by bar/pie.)